cmd/mattermost/commands/channel_test.go

```diff
@@ -39,8 +40,12 @@ func TestRemoveChannel(t *testing.T) {
 	// should fail because channel does not exist
 	require.Error(t, RunCommand(t, "channel", "remove", th.BasicTeam.Name+":doesnotexist", th.BasicUser2.Email))
 
+	time.Sleep(time.Second)
```

Comment:
If this is needed, something is wrong in the code, or in the test.

Reply:
Agreed.